pkg/traceql/engine.go

```diff
@@ -315,7 +303,11 @@ func (e *Engine) asTraceSearchMetadata(spanset *Spanset) *tempopb.TraceSearchMet
 		}
 
 		for attribute, static := range atts {
-			if attribute.Intrinsic == IntrinsicName || attribute.Intrinsic == IntrinsicDuration {
+			if attribute.Intrinsic == IntrinsicName ||
+				attribute.Intrinsic == IntrinsicDuration ||
+				attribute.Intrinsic == IntrinsicTraceDuration ||
+				attribute.Intrinsic == IntrinsicTraceRootService ||
+				attribute.Intrinsic == IntrinsicTraceRootSpan {
```

Comment:
👍 Ah ok I missed this before.  We could probably simplify the Span interface and put the intrinsics just in the map and not dedicated fields in the future.